includes/collection/class-inbox.php

```diff
@@ -88,6 +121,13 @@ public static function add( $activity, $user_id ) {
 			\kses_init_filters();
 		}
 
+		// Add recipients as separate meta entries after post is created.
+		if ( ! \is_wp_error( $id ) && $id > 0 ) {
```

Comment:
```suggestion
		if ( ! \is_wp_error( $id ) ) {
```
I'd expect post Ids to always be more than 0.